src/flows/components/panel/FlowPanel.tsx

```diff
@@ -122,6 +122,9 @@ const FlowPanel: FC<Props> = ({id, children, controls, persistentControl}) => {
 
   return (
     <>
+      {!flow.readOnly && flow.data.indexOf(id) === 0 && (
+        <InsertCellButton id={id} insertAbove={true} />
```

Comment:
feels weird to have a panel that has two insert buttons, and that all of this is decided on the panel's view layer instead of being able to have an insert button _without_ a panel that always inserts at the top (being determined at the flow.list context), but i'm having a hard time articulating why. Like how a panel should be self contained enough to operate outside of a notebook context (but idk how true that is tbh), and this makes the panel now have to know that it exists in a list, and that it has a position in that list